text/0000-clippy-uno.md

```diff
@@ -0,0 +1,263 @@
+   For example, having a method named `into_foo()` that doesn't take `self` by-move
+ - Complexity (Warn): For detecting unnecessary code complexities and helping
+   simplify them. For example, a lint that asks you to replace `.filter(..).next()` with `.find(..)`
+ - Perf (Warn): Detecting potential performance footguns, like using `Box<Vec<T>>` or calling `.or(foo())` instead of `or_else(foo)`.
+ - Pedantic (Allow): Controversial or exceedingly pedantic lints
+ - Nursery (Allow): For lints which are buggy or need more work
+ - Cargo (Allow): Lints about your Cargo setup
+ - Restriction (Allow): Lints for things which are not usually a problem, but may be something specific situations may dictate disallowing.
+ - Internal (Allow): Nothing to see here, move along
+ - Deprecated (Allow): Empty lints that exist to ensure that `#[allow(lintname)]` still compiles after the lint was deprecated.
+
+Lints can only belong to one lint group at a time, and the lint group defines the lint level. There is a bunch of overlap between
+the style and complexity groups -- a lot of style issues are also complexity issues and vice versa. We separate these groups
+so that people can opt in to the complexity lints without having to opt in to Clippy's style.
+
+## Compiler uplift
+
+The compiler has historically had a "no new lints" policy, partly with the desire that lints would
+incubate outside of Clippy. This feels like a good time to look into uplifting these lints.
+
+This RFC does not _yet_ propose lints to be uplifted, but the intention is that the RFC
+discussion will bring up lints that the community feels _should_ be uplifted and we can list them here.
+
+Such an uplift may change the lint level; correctness lints are Deny
+by default in Clippy but would probably switch to Warn if uplifted since the compiler is more
+conservative here (Using clippy is in itself an opt-in to a "please annoy me more" mode).
+
+
+We'd also like to establish a rough policy for future lints here:  Some correctness lints should probably belong in the compiler,
+whereas style/complexity/etc lints should probably belong in Clippy. Lints may be incubated in Clippy, of course.
+
+I don't think the compler will want _all_ correctness lints here, however if the lint is about a common enough situation
+where it being _not_ a bug is an exceedingly rare case (i.e. very low false positive frequency) it should probably belong in the
+compiler.
+
+ [cat]: #lint-categorization
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+
+## Lint categorization
+
+This categorization can be browsed [online].
+
+ [online]: http://rust-lang-nursery.github.io/rust-clippy/current/
+
+Please leave comments on thoughts about these lints -- if their categorization is correct, if they should exist at all, and if we should be uplifting them to the compiler.
```

Comment:
Uhm... the same things seem to be repeated in every category... is this intended?